src/qml/qgismobileapp.qml

```diff
@@ -931,16 +931,35 @@ ApplicationWindow {
     }
   }
 
+  NavigatingDrawer{
```

Comment:
I think we need a more generic name here. I'd name this item `InformationDrawer` instead because not only do we have positioning details (which aren't navigation-centric when you don't see a navigation destination), but we also have sensor data (more on that below ;-) )

Reply:
it makes sense :+1: